src/java/io/bazel/rulesscala/coverage/instrumenter/JacocoInstrumenter.java

```diff
@@ -100,7 +116,7 @@ public FileVisitResult visitFile(Path inPath, BasicFileAttributes attrs) {
             }
 
             private FileVisitResult actuallyVisitFile(Path inPath, BasicFileAttributes attrs) throws Exception {
-                Path outPath = outFS.getPath(inPath.toString());
+                Path outPath = Paths.get(tempDir.toString(), inPath.toString());
```

Comment:
Thanks for noting that. I guess that would be `Path.resolve(...)`.

Reply:
I tried this and failed it:

`Path outPath = tempDir.resolve(inPath.subpath(0, inPath.getNameCount()));`

It throws an Exception: java.nio.file.ProviderMismatchException.
Documentation says "Unchecked exception thrown when an attempt is made to invoke a method on an object created by one file system provider with a parameter created by a different file system provider."
So there may be a conflicts in the filesystems. Can I leave this like in the original version?

Reply:
`tempDir.resolve(inPath)` gives the same Exception.

Reply:
`Path outPath = tempDir.resolve(inPath.subpath(0, inPath.getNameCount()).toString());` would work. The reason is that we need to remove "/" from the beginning and we need to convert the path we add to string to avoid the ProviderMismatchException.

Reply:
Exchanged code to the last proposal.